content/events/2023-10-28_urbit-assembly-lisboa.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Urbit Assembly Lisboa 2023"
+starts = "2023-10-28T10:00:00"
+ends = "2023-10-29T20:00:00"
+timezone = "Europe/Lisbon"
+location = "Lisbon – Portugal"
+image = "https://storage.googleapis.com/media.urbit.org/site/events/urb01_assembly23_std_1-1.png"
+registration_url = ""
+description = "The third Urbit Assembly & first one outside of the US"
+dark = true
+darken_image = "true"
+
++++
+![Screenshot](https://storage.googleapis.com/media.urbit.org/site/events/urb01_assembly23_std_16-9.png)
+
+Urbit is coming to Europe! Save the date for Urbit Assembly 2023 & Urbit Week in Lisbon this year. 
+
+Early-bird tickets are going live by the end of may in the [UF Public Group](/groups/~halbex-palheb/uf-public). 
+
+This year's Assembly will be a conference of two days filled with presentations of all companies on Urbit followed by a week of side-events throughout Lisbon. Don’t miss it.
```

Comment:
Change to this, then good to go: "We’re changing up the format this year: the first two days will be more “conference-like” with launches, presentations and panels from the Urbit ecosystem.

Afterwards we’ll have a week of side events hosted by the community throughout Lisbon.

You’re not going to want to miss it."